superduperdb/db/mongodb/cdc.py

```diff
@@ -703,7 +703,11 @@ def listen(
             self._startup_event.wait(timeout=timeout)
         except Exception:
             logging.error('Listening service stopped!')
-            s.CFG.cluster.cdc = False
+            try:
```

Comment:
ah, sorry, do this instead:

`CDC_COLLECTION_LOCKS.pop(self._on_component.name, None)`

Reply:
il do it

Reply:
done, please check and let me know @rec